app/safe/internal/server/route/delete.go

```diff
@@ -40,6 +41,11 @@ func isSentinel(j audit.JournalEntry, cid string, w http.ResponseWriter, svid st
 }
 
 func Delete(cid string, w http.ResponseWriter, r *http.Request, svid string) {
+	if env.SafeManualKeyInput() && !state.MasterKeySet() {
```

Comment:
If keys will be entered manually but they are not there, then deny all API interaction until the keys are there.